examples/plot_3d_and_4d_niimg.py

```diff
@@ -0,0 +1,108 @@
+"""
+3D and 4D niimgs: handling and visualizing
+==========================================
+
+Here we discover how to work with 3D and 4D niimgs.
+"""
+
+###############################################################################
+# Downloading tutorial datasets from Internet
+# --------------------------------------------
+#
+# Nilearn comes with functions that download public data from Internet
+#
+# Let's first check where the data is downloaded on our disk:
+from nilearn import datasets
+print(datasets.get_data_dirs())
```

Comment:
I know that this is example but it would be nice, IMHO, to label a bit more what is printed:
`print('Datasets are stored in: ' + datasets.get_data_dirs())`